spec/javascripts/Plans/components/DefaultPlanSelectCard.spec.jsx

```diff
@@ -1,75 +1,82 @@
+  const wrapper = mountWrapper()
   expect(wrapper.exists()).toBe(true)
 })
 
-it.todo('should have a "no default plan" option')
+it('should have a helper text', () => {
+  const wrapper = mountWrapper()
+  
+  const helperText = wrapper.find('.pf-c-helper-text')
 
-it('should show a success message if request goes well', async () => {
-  ajax.mockResolvedValue({ ok: true })
-  const wrapper = mount(<DefaultPlanSelectCard {...props} />)
+  expect(helperText.exists()).toBe(true)
+})
 
-  await act(async () => {
-    wrapper.find('DefaultPlanSelect').invoke('onSelectPlan')(plan)
-  })
+it('should have a no default plan inside plans', () => {
+  const wrapper = mountWrapper()
+  openSelect(wrapper)
 
-  expect(noticeSpy).toHaveBeenCalledWith('Default plan was updated')
+  //Check what to do here
 })
 
-it('should show an error message when selected plan does not exist', async () => {
-  ajax.mockResolvedValueOnce({ status: 404 })
-  const wrapper = mount(<DefaultPlanSelectCard {...props} />)
+// TODO: Fix this test
+it('should be able to select a plan', () => {
+  const wrapper = mountWrapper()
+  openSelect(wrapper)
+  wrapper.find('SelectOption button').first().simulate('click')
+
+  const selected: SelectOptionObject = wrapper.find('Select#id').prop('selections')
+  expect(selected.name).toBe('(No default plan)')
+})
 
-  await act(async () => {
-    wrapper.find('DefaultPlanSelect').invoke('onSelectPlan')(plan)
-  })
+it('should have a disabled button', () => {
+  const wrapper = mountWrapper()
+  wrapper.find('.pf-c-select__toggle-clear').simulate('click')
 
-  expect(errorSpy).toHaveBeenCalledWith("The selected plan doesn't exist.")
+  expect(wrapper.find('button[type="submit"]').prop('disabled')).toBe(true)
 })
 
-it('should show an error message when server returns an error', async () => {
-  ajax.mockResolvedValue({ status: 403 })
-  const wrapper = mount(<DefaultPlanSelectCard {...props} />)
+it('should enable the button when a plan is selected', () => {
+  const wrapper = mountWrapper()
+  wrapper.find('.pf-c-select__toggle-clear').simulate('click')
+  expect(wrapper.find('button[type="submit"]').prop('disabled')).toBe(true)
 
-  await act(async () => {
-    wrapper.find('DefaultPlanSelect').invoke('onSelectPlan')(plan)
-  })
+  openSelect(wrapper)
+  wrapper.find('SelectOption button').first().simulate('click')
 
-  expect(errorSpy).toHaveBeenCalledWith('Plan could not be updated')
+  expect(wrapper.find('button[type="submit"]').prop('disabled')).toBe(false)
 })
 
-it('should show an error message when connection fails', async () => {
-  // $FlowExpectedError[cannot-write] suppress error logs during test
-  console.error = jest.fn()
-
-  ajax.mockRejectedValue()
-  const wrapper = mount(<DefaultPlanSelectCard {...props} />)
+it('should disable the plan already selected', () => {
+  const wrapper = mountWrapper()
+  const option = () => wrapper.find('SelectOption button').findWhere(n => n.text() === 'My Plan').first()
```

Comment:
Avoid using hardcoded values when possible:
```suggestion
  const option = () => wrapper.find('SelectOption button').findWhere(n => n.text() === plan.name).first()
```